### PR TITLE
New article: How I contribute to Rust Compiler

### PR DESCRIPTION
**Pull request by chenyukang**

### CODE REVIEW

**Review by Folyd — COMMENTED** [review]
Thanks for the contributions. Just one nit-pick, don't forget to update the `zine.toml` files. Please see https://rustmagazine.github.io/contribution/#add-your-article.

File: content/issue-1/contribute-to-rustc.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# I Learn Rust By Contributing To Rust Compiler
+
+---
```
Comment (Folyd) [inline content/issue-1/contribute-to-rustc.md:3]: We can delete those lines. BTW, I suggest renaming the title to "How I contribute to Rust Compiler". Any thoughts? @LCrossman @macko76

File: content/issue-1/zine.toml [hunk 2]
```diff
@@ -12,3 +12,14 @@ pub_date = "2022-12-10"
...
 featured = true
 i18n.zh = { path = "/announcing-zh", file = "announcing-zh.md", title = "宣布 Rust 杂志", author = ["folyd", "rust-magazine"] }
+
+
+[[article]]
+path = "/contribute-to-rustc"
+file = "contribute-to-rustc.md"
+title = "I Learn Rust By Contributing To Rust Compiler"
+author = "yukang"
+topic = ["optional-topic"]
```
Comment (Folyd) [inline content/issue-1/zine.toml:22]: `optional-topic` is a placeholder, I'll give it a concrete one later.

File: content/issue-1/contribute-to-rustc.md (hunk 1 above)
Comment (chenyukang) [reply content/issue-1/contribute-to-rustc.md:3]: yes, I didn't come with a good subject, any better one?

**Pull request merged**